wolox-books/src/app/screens/auth/screens/book-list/components/book-card/book-card.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../../../../../scss/variables.scss';
+@import '../../../../../../../scss/colors.scss';
+
+.book-card {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 10px;
+  padding: 20px;
+  width: 250px;
+}
+
+.book-img {
+  height: 200px;
+  margin-bottom: 20px;
+  width: 200px;
```

Comment:
use only height or width to not deform the image, or use an image container and the object-fit property